watchtower-plugin/src/main.rs

```diff
@@ -65,14 +67,23 @@ async fn register(
         },
     );
 
-    let mut tower_net_addr = format!("{}:{}", host, port);
-    if !tower_net_addr.starts_with("http") {
-        tower_net_addr = format!("http://{}", tower_net_addr)
-    }
+    let tower_net_addr = {
+        if !host.starts_with("http") {
+            host = format!("http://{}", host)
+        }
```

Comment:
nit: `host` isn't quite the name for an HTTP url.
Also let's check `.starts_with("http://")` because a host name might actually start with `http` :laughing:.